core/runtime/src/main/java/io/quarkus/runtime/logging/FileConfig.java

```diff
@@ -48,7 +53,7 @@
     /**
      * The file logging log level
```

Comment:
OK, so this is not related to your patch but this comment is obviously incorrect. Let's fix it?

Reply:
Done.